Integrations/Tenable_sc/Tenable_sc.yml

```diff
@@ -165,16 +186,33 @@ script:
       type: string
     - contextPath: TenableSC.ScanResults.Vulnerability.PluginDetails
       description: Vulnerability plugin details
+      type: Unknown
     - contextPath: CVE.ID
       description: CVE ID
-    description: 'Get details about a given vulnerability from a given Tenable.sc
-      scan '
-  - name: tenable-sc-get-scan-status
-    arguments:
-    - name: scan_results_id
-      required: true
+      type: Unknown
+    - contextPath: TenableSC.ScanResults.Vulnerability.Hosts.IP
```

Comment:
Should it be "Hosts" or "Host"? "Vulnerability" or "Vulnerabilities"? I'm not sure if the default in Demisto is plural or singular (Itay Keren will know) but we should in any case remain consistent.
This also holds true to all other added context paths